apis/Google.Cloud.Language.V1/Google.Cloud.Language.V1.IntegrationTests/SmokeTest.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ClientTesting;
+using Grpc.Core;
+using Xunit;
+
+namespace Google.Cloud.Language.V1.IntegrationTests
+{
+    public sealed class SmokeTest : CloudProjectFixtureBase
```

Comment:
Rather than just `SmokeTest` (which are usually handled in a SmokeTests project or via a `smoketests.json` file), could you specify what it is you're trying to test with this? We already have a `smoketests.json` file that does what I'd normally consider a smoke test: does the library work at all, can it make a successful request.

It looks like you're really trying to test the error handling and defaulting - in which case I think it would be clearer to say so.